sockeye/evaluate.py

```diff
@@ -41,6 +40,15 @@ def raw_corpus_bleu(hypotheses: Iterable[str], references: Iterable[str], offset
     """
     return sacrebleu.raw_corpus_bleu(hypotheses, [references], smooth_floor=offset).score / 100
 
+def raw_corpus_chrf(hypotheses: Iterable[str], references: Iterable[str]) -> float:
+    """
+    Simple wrapper around sacreBLEU's chrF implementation, without tokenization.
+
+    :param hypotheses: Hypotheses stream.
+    :param references: Reference stream.
+    :return: chrF score as float between 0 and 1.
+    """
+    return sacrebleu.corpus_chrf(hypotheses, references, remove_whitespace=True)
```

Comment:
lets make the default order and beta parameters explicit here: `order=sacrebleu.CHRF_ORDER` etc.

Reply:
Will do.